scripts/helpers/check_zap_against_xml.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import sys
+import xml.etree.ElementTree as ET
+
+import click
+
+DATA_MODEL_PATH = 'data_model/clusters'
+
+
+@click.command()
+@click.option(
+    '--zap',
+    required=True,
+    type=str,
+    help='ZAP file')
+@click.option(
+    '--matter-root',
+    required=True,
```

Comment:
could we default using the path of `__file__` like: `os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))` or something like that

Reply:
done.